notebooks/SVM.ipynb

```diff
@@ -0,0 +1,688 @@
+{
```

Comment:
<p>Looks like this is an old file.  plz delet. <code>git rm &lt;file&gt;</code>  is what you want.  Also, rename the new notebook so it doesn't have the date in the name.  <code>git mv &lt;old_filename&gt; &lt;new_filename&gt;</code>  should do it</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/srensi/tmprss2/pull/25/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>